.github/actions/build-component-per-arch/main.js

```diff
@@ -62,6 +62,9 @@ async function shell_cmd(cmd) {
                 core.setFailed(`Failed with unknown platform: ${core.getInput('platform')}`)
                 return
         }
+        
+        process.env.AGENT_FEATURES = "agent-full akri-udev akri-opcua akri-onvif"
```

Comment:
It seems like `build_rust` is never being set to `1` now that we are building all the code once. should we remove the optionality of building rust here or leave it in case we toggle back. If we dont think it is necessary to leave in anymore, then this env var does not need to be set

Reply:
seems fine to remove it

Reply:
I removed what was added by this PR (the `AGENT_FEATURES`...) and in a separate PR we can remove the unnecessary build logic